sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/DigitalTwinsLifecycleAsyncSample.java

```diff
@@ -333,4 +332,48 @@ public static void connectTwinsTogether() throws IOException, InterruptedExcepti
         // Wait until the latch count reaches zero, signifying that the async calls have completed successfully.
         connectTwinsLatch.await(MAX_WAIT_TIME_ASYNC_OPERATIONS_IN_SECONDS, TimeUnit.SECONDS);
     }
+
+    /**
+     *
+     * Queries for all the digital twins.
+     * @throws IOException If an I/O error is thrown when accessing the starting file.
+     * @throws InterruptedException If the current thread is interrupted while waiting to acquire latch.
+     */
+    public static void queryTwins() throws InterruptedException {
+        System.out.println("QUERY DIGITAL TWINS");
```

Comment:
We've been using the `ConsoleLogger` helper class for printing console statements.

Reply:
Are you sure? I say  System.out.println all across the file.

Reply:
That's weird, are you sure you have the latest code?

Reply:
Looks like it has changed after I rebased. Let me switch it.